ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/WebSocketSubscriber.java

```diff
@@ -98,7 +99,7 @@ public void onComplete() {
   public void onSchema(Schema schema) {
     try {
       session.getBasicRemote().sendText(
-          mapper.writeValueAsString(schema)
+          mapper.writeValueAsString(EntityUtil.buildSourceSchemaEntity(schema))
```

Comment:
where do we draw the line internally on when we use connect `Schema`s vs. where we use `SchemaInfo` schemas? Is it only for the rest APIs or do we also use `SchemaInfo` internally?

Reply:
Its just for the API.